singles/base/pom.xml

```diff
@@ -48,6 +62,10 @@
                                         <include>it/unimi/dsi/fastutil/objects/ObjectCollection$*.class</include>
                                         <include>it/unimi/dsi/fastutil/objects/ObjectIterator.class</include>
                                         <include>it/unimi/dsi/fastutil/objects/ObjectIterator$*.class</include>
+                                        <include>it/unimi/dsi/fastutil/shorts/ShortConsumer.class</include>
```

Comment:
rather move these to short base

Reply:
why are these even in this PR anyways?

Reply:
intbase, longbase, etc rely on these classes now, apparently. I included them in base since every other "base" seem to rely on them.